config.yml

```diff
@@ -2,89 +2,48 @@ version: 1
 build:
   main: cmd/poktrolld
 accounts:
-  - name: faucet
-    mnemonic: "baby advance work soap slow exclude blur humble lucky rough teach wide chuckle captain rack laundry butter main very cannon donate armor dress follow"
+  - name: faucet # Faucet - initial funds we need to test the network and bootstrap the migration from Morse.
```

Comment:
Do we want/need a faucet account for mainnet?

Reply:
No, but we need an account that has initial funds - enough to bootstrap the network and accommodate migration from Morse. It can be grove or dao account instead.